threadit-web/src/stores/ThreadStore.ts

```diff
@@ -0,0 +1,20 @@
+import { makeObservable, action, observable } from "mobx";
+import { IThread } from '../models/Thread';
+import ThreadAPI from "../api/ThreadAPI";
+
+export class ThreadStore {
+    @observable
+    _thread?: IThread = undefined;
```

Comment:
do we need to store the thread for any reason?

Reply:
Nope, good call